.github/workflows/ci.yml

```diff
@@ -2,8 +2,10 @@ name: CI
 on:
   push:
     branches: [master]
-    tags: ["*"]
+    tags: [v*]
   pull_request:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Do we need this scheduled ci? I was overlooking these lines in the previous review.

Reply:
Need? No. Sometimes, though, changes to dependencies or Julia itself are made between PRs, causing the CI to fail inexplicably when someone next goes to open a PR. This mitigates that. Quaternions is simple enough that this is not likely though, but it doesn't harm to run the CI once a day.

Reply:
Thanks for the quick response! I just thought updating `gh-pages` every day is a little too frequent.